pkg/model/serializer.go

```diff
@@ -722,6 +722,63 @@ func checkFileAndNotDirectory(path string) error {
 	return fmt.Errorf("Secret '%s' is not a regular file", path)
 }
 
+type hybridModeInfo struct {
```

Comment:
would it make sense to move the hybrid "models" and variables to a pkg `hybrid` in order to avoid making model bigger? 

cc @jLopezbarb

Reply:
@teresaromero I was thinking about the same, like we did for `pkg/cache`, howerer the Hybrid type depends on types from `model` that are shared with `Dev` so it turns into a cycle dependency so I gave up moving only Hybrid, I think once we move the whole "Dev" type outside of the model it will be easier

Reply:
![](https://media.giphy.com/media/bUkXxGkGYb5bq/giphy.gif)